library/docs/changelog.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Upgrade to DataStore `1.0.0-rc01`.
 - Upgrade to AGP `4.2.2`.
 - Upgrade to Hyperion `0.9.33`.
+- Update `Sources` section in the user guide
```

Comment:
There's no need to update change log with documentation changes.

Reply:
@MiSikora Removed and rebased